src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -3,9 +3,9 @@
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<span class="flc-prefsEditor-speak-descr"></span>
         <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
             <span class="fl-prefsEditor-switch-inner"></span>
         </span>
-        <span class="flc-prefsEditor-speak-choice-label"></span>
```

Comment:
Gotcha, I think this change is in the [FLUID-5678](https://github.com/fluid-project/infusion/pull/611) pull request